### ISSUE TITLE
[Bug]: Incorrect FPS value calculation

### ISSUE THREAD

**Issue by Hellisotherpeople** [continue-revolution/sd-webui-animatediff#25]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

Have to set FPS to say, 200 to get results consistent with everyone else. The default settings are laughably wrong. 

See this reddit comment thread, I've tested the code fix and it works, so port over what the comments recommend: https://www.reddit.com/r/StableDiffusion/comments/152n2cr/comment/jsfuuva/?utm_source=share&utm_medium=web2x&context=3

### Steps to reproduce the problem

Run extension with default settings. Gif will be ultra slow

### What should have happened?

Gif should run at normal speed

### Commit where the problem happens

Latest

### What browsers do you use to access the UI ?

Mozilla Firefox

### Command Line Arguments

```Shell
Doesn't matter
```

### Console logs

```Shell
Doesn't matter
```

**Comment by gregcman** [comment]
Can confirm

**Comment by continue-revolution** [comment]
FPS issue shuold have been resolved via #53 

If problem still exists, feel free to let me know

**Comment by p3p-pixel** [comment]
The speed of the generated GIF did not change even if I changed the fps input value, so I modified the source code as follows. I confirmed that the speed changed after that.
The calculation formula is from an error message that occurred when I tried inputting the fps argument.
```
# imageio.mimsave(video_path, video_list, duration=(1/fps), loop=loop_number)
imageio.mimsave(video_path, video_list, duration=(1000 * 1 / fps), loop=loop_number)

# When I tried inputting the fps argument.
TypeError: The keyword `fps` is no longer supported. Use `duration`(in ms) instead, e.g. `fps=50` == `duration=20` (1000 * 1/50).
```
